forge-gui/res/cardsfolder/k/kitsune_riftwalker.txt

```diff
@@ -2,6 +2,6 @@ Name:Kitsune Riftwalker
 ManaCost:1 W W
 Types:Creature Fox Wizard
 PT:2/1
-K:Protection from Spirits
-K:Protection from Arcane
+K:Protection:Spirit
```

Comment:
`Cephalid Snitch` is the problematic one, because it says:
> Sacrifice Cephalid Snitch: Target creature loses protection from black until end of turn.

this can cause "Protection from all colors" to split into Protection from the other 4"